### ISSUE TITLE
Parallax with ListView

### ISSUE THREAD

**Issue by joshbedo** [oblador/react-native-parallax#13]
Do these work together? The parallax effect appears to stop working when i add parallax within the `renderRow` method.

**Comment by oblador** [comment]
Not really, see https://github.com/oblador/react-native-parallax/issues/5. I'd be happy to accept a PR if you can solve the problem :+1:
